src/app/products/products.component.ts

```diff
@@ -7,4 +20,30 @@ import { Component } from '@angular/core';
 })
 export class ProductsComponent {
 
+  alldata : productsData[] = [];
+  errorFlag=false;
+  constructor(private ds:DataShareService){
+
+  
+
+  // loadProductData(){
+    
+  this.ds.callGetData().subscribe( 
+    
+    (productResult : any) => {debugger;
```

Comment:
Do not add debugger statement in code and commit

Reply:
fixed this and commited changes